js/index.js

```diff
@@ -1,5 +1,5 @@
 // navbar active
-var navbarButton = document.querySelectorAll('.svg__imagee');
+const navbarButton = document.querySelectorAll('.svg__imagee');
 
 navbarButton.forEach(function (button) {
```

Comment:
Everytime a new link is opened, this is repeated, hence the state is not saved and active status is not shown. Instead of adding class on click event, I suggest you do something like:

if(window.location.pathname ==='/index.html' || window.location.pathname ==='/'){
    navbarButton[0].classList.add('navbar__icons-svg-active')
};

Reply:
fixed

Reply:
or simply you can can add active class for current page nav item tag, instead of manually adding it from Javascript.
"The Best Code is No Code At All" - 10X Developers